node/src/main/scala/coop/rchain/node/grpc.scala

```diff
@@ -78,13 +86,18 @@ object GrpcServer {
 
     def exec(reader: Reader): Future[ReplResponse] =
       Task
-        .delay(buildNormalizedTerm(reader))
+        .coeval(buildNormalizedTerm(reader))
+        .attempt
         .flatMap {
           case Left(er) =>
-            Task.now(s"Error: $er")
+            er match {
+              case _: InterpreterError => Task.now(s"Error: ${er.toString}")
+              case th: Throwable       => Task.now(s"Error: $th")
+            }
           case Right(term) =>
             evaluate(runtime.reducer, term).attempt.map {
-              case Left(ex) => s"Caught boxed exception: $ex"
+              case Left(ie: InterpreterError) => s"Error: ${ie.toString}"
+              case Left(ex)                   => s"Caught boxed exception: $ex"
```

Comment:
@rabbitonweb @sebtomba you can have a look at those changes as they affect your code `grpc.scala`. `buildNormalizedTerm` returns `Coeval` now so I had to call `attempt` to raise all the possible errors to `Either`. Also I've introduced the `InterpreterError` that is a supertype for all possible errors in the rholang interpreter. They are overriding `toString` method with better messages.